contributors/devel/api_changes.md

```diff
@@ -479,7 +508,30 @@ use our own equivalents for JSON serialization. The `pkg/api/serialization_test.
 will verify that your protobuf serialization preserves all fields - be sure to
 run it several times to ensure there are no incompletely calculated fields.
 
-## Edit json (un)marshaling code
+### Generate Clientset
+
+See [this document](generating-clientset.md) specific to generating client sets.
+
+### Generate Listers
+
+`lister-gen` is a tool to generate listers for a client. It requires a
+client generated by `client-gen`.
+
+`lister-gen` requires the `// +genclient=true` annotation on each
+exported type in both the unversioned base `types.go` as well as each
+specifically versioned `types.go`.
+
+`lister-gen` requires the `// +groupName=` annotation on the `doc.go` in
```

Comment:
There are a couple more annotations:

```go
// +k8s:defaulter-gen=TypeMeta
// +k8s:deepcopy-gen=package,register
```

That are not doc'd anywhere that I'm aware of - will you doc those as well?